### PR TITLE
Add get graph models to Faros client

### PR DESCRIPTION
**Pull request by ypc-faros**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

**Review by ted-faros — APPROVED** [review]
Suggested a version assertion, if possible.

File: src/client.ts [hunk 1]
```diff
@@ -89,6 +90,15 @@ export class FarosClient {
     }
   }
 
+  async models(graph: string): Promise<ReadonlyArray<Model>> {
```
[the remark this answers is not in the document]
Comment (ypc-faros) [reply src/client.ts:102]: Done, thanks!!!

[the remark this answers is not in the document]
Comment (tovbinm) [reply src/client.ts:102]: remove newline from the error
```
 throw new VError(`listing models is not supported for ${this.graphVersion} graphs`);
```

[the remark this answers is not in the document]
Comment (ypc-faros) [reply src/client.ts:102]: Thanks! https://github.com/faros-ai/faros-js-client/pull/29

**Pull request merged**